src/pages/Banner/Banner.module.css

```diff
@@ -1,44 +1,43 @@
 .Banner {
-    position: relative;
-    display: block;
-    text-align: center;
+  position: relative;
+  display: block;
+  text-align: center;
 }
 
-
 .Banner h1 {
-    font-size: 70px;
-    font-weight: bold;
-    position: absolute; 
-    top: 15%;
-    left: 0;
-    right: 0;
-    z-index: 1;
+  font-size: 70px;
```

Comment:
you could also consider using rem for relative font size, this can come in handy when the page is zoomed in/out by the user